gnome-shell/src/gnome-shell-sass/_common.scss

```diff
@@ -157,15 +152,6 @@ StScrollBar {
   color: lighten($bg_color, 4%);
   &:hover { color: $_hover_bg_color; }
   &:active { color: $_active_bg_color; }
-
```

Comment:
I think we need to keep these lines

Reply:
Yes sorry, last rebase made a mess

Reply:
Hey no problem, sometimes all those different branches and versions makes us dizzy :D